image-classification/download_dataset_pod.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mlperf-image-classification-download-dataset
+spec:
+  restartPolicy: Never
+  containers:
+    - name: worker
+      image: registry.tensorstack.cn/t9k/mlperf-nvidia:resnet50
+      args:
+        - sleep
+        - "1000000"
+      resources:
+        limits:
```

Comment:
1. is 1 CPU sufficient? it's `limits`, a larger value is usually more flexible. you can add a `requests` node and use a smaller value for `requests` but much larger ones for `limits`.
2. is GPU required for data preparation?